### PR TITLE
Register Subdomain: gauravkulkarni

### PR DESCRIPTION
**Pull request by gkulk007**
Added the domain: gauravkulkarni

### CODE REVIEW

File: domains/gauravkulkarni.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "gkulk007",
+        "email": "gaurav.kulkarni@mescoeorg.onmicrosoft.com"
+      },
+
+      "record": {
+        "CNAME": "www"
```
Comment (WilliamDavidHarrison) [inline domains/gauravkulkarni.json:9]: Invalid CNAME

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/gauravkulkarni.json (hunk 1 above)
Comment (gkulk007) [reply domains/gauravkulkarni.json]: updated CNAME

**Review by wdhdev — CHANGES_REQUESTED** [review]
I cannot reach your website.

File: domains/gauravkulkarni.json (hunk 1 above)
Comment (gkulk007) [reply domains/gauravkulkarni.json]: Updated the settings and removed the previous domain. Please check once again. Thankyou

**Review by wdhdev — APPROVED** [review]
Nice website!

**Comment by gkulk007** [comment]
> Nice website!

Thank you so much 🥳🥳

**Review by andrewstech — APPROVED** [review]

**Pull request merged**